cognite/extractorutils/base.py

```diff
@@ -193,13 +195,23 @@ def __enter__(self) -> "Extractor":
         Returns:
             self
         """
+
+        # Environment Variables
+        self.env_file_path = find_dotenv()
+        if len(self.env_file_path) > 0:
+            load_dotenv(dotenv_path=self.env_file_path, override=True)
+            self.dotenv_message = f"Successfully ingested environment variables from\n{self.env_file_path}"
+        else:
+            self.dotenv_message = "Could not ingest environment variables from .env file"
```

Comment:
I suggest a slight rewording of this message, now it seems like something went wrong, while it's just that no env file exists (which I suspect might be the case for quite a few).

Maybe something like `No .env file located`?

Reply:
Also, if this message is only used here I suggest just using a local variable instead of assigning a new attribute (so no `self`) 🙂